packages/apollos-data-connector-rock/src/content-items/data-source.js

```diff
@@ -273,7 +273,8 @@ export default class ContentItem extends RockApolloDataSource {
       .tz(ROCK.TIMEZONE)
       .format()
       .split(/[-+]\d+:\d+/)[0];
-    return `((StartDateTime lt datetime'${date}') or (StartDateTime eq null)) and ((ExpireDateTime gt datetime'${date}') or (ExpireDateTime eq null)) `;
+    const filter = `(((StartDateTime lt datetime'${date}') or (StartDateTime eq null)) and ((ExpireDateTime gt datetime'${date}') or (ExpireDateTime eq null))) and Status eq 2}`;
```

Comment:
I think this should be 
```suggestion
    const filter = `(((StartDateTime lt datetime'${date}') or (StartDateTime eq null)) and ((ExpireDateTime gt datetime'${date}') or (ExpireDateTime eq null))) and Status eq 'Approved'}`;
```

I found out yesterday that Rock expects a string for this field :😅

Reply:
I thought the values we looked at were integers - well, still strings, but strings that represent integers.

Reply:
They are. But rock expects strings when you use the `$filter`. I only know this by playing around with the API console, try going to the `ContentChannelItems` endpoint in the Rock API browser and trying the following in the `$filter` field.

`Status eq 2`

`Status eq 'Approved'`

Reply:
Oh, okay; so `Status eq '2'` would work? Or is it preferable to use the human-readable string?

Reply:
I don't think `Status eq '2'` would work, but you can test in the Rock API console :)

Reply:
Looks like it works okay, but I'm just going to leave it with the word so its clearer.